src/kernel/clock.c

```diff
@@ -35,3 +35,19 @@ void clock_adjust(timestamp wallclock_now, s64 temp_cal, timestamp sync_complete
     rtc_settimeofday(sec_from_timestamp(wallclock_now));
 }
 KLIB_EXPORT(clock_adjust);
+
+closure_function(0, 1, boolean, timer_id_rtc,
+                timer, t)
+{
+    return t->id == CLOCK_ID_REALTIME || t->id == CLOCK_ID_REALTIME_ALARM;
```

Comment:
Should include also CLOCK_ID_REALTIME_COARSE

Reply:
Thanks, got it